src/main/scala/dpla/ingestion3/enrichments/VocabEnforcer.scala

```diff
@@ -140,26 +139,50 @@ object DcmiTypeStringMapper extends VocabEnforcer[String] {
 }
 
 /**
-  * Reads in ISO-693-3 data from tab-delimited file and generates a
+  * Reads in iso693-X data from CSV files and generates a
   * map to reconcile language abbreviations to the full term
   *
   */
 object LanguageMapper extends VocabEnforcer[String] {
+
+  private val languageFiles = Seq(
```

Comment:
This will not work if the project is ever bundled into a JAR. Basically everything in src/main/resources gets bundled into the top level directory of the JAR. 

The better way to do this is to use the classpath/classloader to locate the file. Unfortunately I don't know off the top of my head how to do that idiomatically in Scala (I know the Java version).

Reply:
Yeah I fought with that for about two hours this morning before giving up and just trying to get done. Insanely frustrating.